lib/activity/comments.js

```diff
@@ -2,14 +2,24 @@ const cache = require('../cache');
 const { Comment } = require('../messages/comment');
 
 module.exports = async (context, subscription, slack) => {
-  const { comment, issue, repository } = context.payload;
+  const { comment, repository } = context.payload;
 
-  // Get rendered message
-  const message = {
-    attachments: [
-      new Comment({ comment, issue, repository }).getRenderedMessage(),
-    ],
-  };
+  let message;
+  if (context.event === 'pull_request_review_comment') {
+    const { pull_request } = context.payload;
+    message = {
+      attachments: [
+        new Comment({ comment, issue: pull_request, repository }).getRenderedMessage(),
+      ],
+    };
+  } else {
+    const { issue } = context.payload;
+    message = {
+      attachments: [
+        new Comment({ comment, issue, repository }).getRenderedMessage(),
+      ],
+    };
+  }
```

Comment:
There's a lot of duplication here just to grab a different variable. What about this instead:

```js
  const issue = context.payload.issue || context.payload.pull_request;
  const message = {
    attachments: [
      new Comment({ comment, issue, repository }).getRenderedMessage(),
    ],
  };
```